tactile_merger/CMakeLists.txt

```diff
@@ -10,30 +10,30 @@ find_package(Eigen3 REQUIRED)
 catkin_package(
   INCLUDE_DIRS include
   LIBRARIES ${PROJECT_NAME}
-#  CATKIN_DEPENDS other_catkin_pkg
+  CATKIN_DEPENDS tactile_msgs
 #  DEPENDS system_lib
 )
 
 include_directories(
-  include/tactile_merger
+  include
   ${catkin_INCLUDE_DIRS}
   ${EIGEN3_INCLUDE_DIR}
 )
 
-add_library(lib${PROJECT_NAME}
+add_library(${PROJECT_NAME}
   src/taxel.cpp
   src/taxel_group.cpp
   src/merger.cpp
 )
-target_link_libraries(lib${PROJECT_NAME} ${catkin_LIBRARIES} tactile_filters)
-set_target_properties(lib${PROJECT_NAME} PROPERTIES OUTPUT_NAME ${PROJECT_NAME})
+target_link_libraries(${PROJECT_NAME} ${catkin_LIBRARIES} tactile_filters)
+set_target_properties(${PROJECT_NAME} PROPERTIES OUTPUT_NAME ${PROJECT_NAME})
 
-add_executable(${PROJECT_NAME} src/merger_node.cpp)
-target_link_libraries(${PROJECT_NAME} lib${PROJECT_NAME})
+add_executable(merger_node src/merger_node.cpp)
```

Comment:
I would suggest to do the rename in a separate PR since it will affect our launch files in other packages, whereas the rest of the PR won't.
and since we will have to edit our wiki doc + launch files, I would also like to take the opportunity to rename this project or at least the node to something that is not called **merger**.  

The reason is that during recent explanation/tutorial to a colleague, it was very confusing to tell that the _tactile_state_publisher_ does "merging" of the various sources of _tactile_states_ into a single topic, that is than fed into a tactile_merger... that does not do merging exactly, more a conversion of group of taxel data to another type. I would prefer some wording including "convert" (since in converts from tactile_state to tactile_contact_state) or aggregate.

Reply:
New name: `contact_force_estimator`?